tools/perf_analysis/plot_pointwise_data.py

```diff
@@ -0,0 +1,132 @@
+#                 plot_pointwise_data.py
+#                 Author:  Brian Pickrell, AMD copyright 2022
+
+#   ingest the JSON files created by  test_broadcast_perf.py
+#   and print optimum local/global inputs for the test set.
+#   Use this code as an example to make pyplot charts or
+#   otherwise do follow-up analysis of the data.
+#
+#   syntax:  python3 plot_pointwise_data.py <input JSON file>
+
+from matplotlib import pyplot as plt
+import json
+import os
+import sys
+import numpy as np
+import pandas
```

Comment:
I suggest importing argparse and adding more flags since currently there are hardcoded values which the user may want to change (such as in L82, L92). You can see tools/roctx.py to see how it's done there. It's fine to have default values with the flags.

Reply:
I've considered this but I don't think it's appropriate here for two reasons:
- parameterizing code (such as reading values from an args list here) without being sure the code will be heavily reused adds a layer of overhead that makes it harder, not easier, to update for very casual use;
- the code in question has a strong coupling between logic and the hard-coded numbers.  It's unlikely anyone would want to run this script with a variety of workitems values without editing the boolean expression they're embedded in as well.

Finally, there are so many good arg parser tutorials available that adding it just for the sake of an example would not add much value.

If and when someone decides to use these scripts on a regular basis with a variety of input values, then it will be time to add arg parser functionality.  I've declared arg parsers in both scripts but for now, they're just placeholders.